acceptance/helper.rb

```diff
@@ -17,7 +17,7 @@ module PuppetDBExtensions
   # by default.
   LeinCommandPrefix = "cd #{GitReposDir}/puppetdb; LEIN_ROOT=true LEIN_SNAPSHOTS_IN_RELEASE=true PATH=/opt/puppetlabs/puppet/bin:$PATH"
 
-  def self.initialize_test_config(options, os_families)
+  def self.initialize_test_config(options, os_families, use_nightlies)
```

Comment:
@wkalt I'm thinking this might make more sense if we add `use_nightlies` to the options hash. I'm digging more into that now